src/main/java/org/opentestsystem/ap/ims/rest/PropertiesApi.java

```diff
@@ -21,7 +20,8 @@ public PropertiesApi(final ItemBankProperties itemBankProperties) {
     }
 
     @GetMapping("/itembankviewer/{itemId}")
-    public String getItemBankViewerUrl(@PathVariable final String itemId, @QueryParam("type") final String type) {
+    public String getItemBankViewerUrl(@PathVariable final String itemId,
+                                       @RequestParam(value = "type", required = false) final String type) {
```

Comment:
Since this isn't required and isn't used, i'd rather just remove it. Is there legacy behavior that wants to set it?

Reply:
I think it can be removed but we should confirm how the TIMS UI is currently calling it.  The format pattern of the  URL must have changed and I vaguely recall that.

We are in the middle of removing the item bank viewer and replacing it with the TIMS scoring service.  This method will go away completely when that happens.